hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableSink.java

```diff
@@ -95,6 +95,9 @@ public SinkRuntimeProvider getSinkRuntimeProvider(Context context) {
         DataStream<Object> pipeline = Pipelines.append(conf, rowType, dataStream, context.isBounded());
         if (OptionsResolver.needsAsyncClustering(conf)) {
           return Pipelines.cluster(conf, rowType, pipeline);
+        } else if (OptionsResolver.isLazyFailedWritesCleanPolicy(conf)) {
```

Comment:
Should we also modify `HoodieFlinkStreamer` here?

Reply:
@stream2000, thanks for the reminder. I have modified `HoodieFlinkStreamer`.